ci_framework/roles/cifmw_cephadm/tasks/apply_spec.yml

```diff
@@ -0,0 +1,62 @@
+---
+# Copyright 2021 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Stat spec file on bootstrap node
+  stat:
+    path: "{{ item }}"
+  register: cifmw_cephadm_spec_files_stat
+  become: true
+  loop:
+    - "{{ cifmw_cephadm_spec }}"
+
+- name: Fail if spec file is missing
+  fail:
+    msg: "{{ item.invocation.module_args.path }} does not exist"
+  loop: "{{ cifmw_cephadm_spec_files_stat.results | list }}"
+  when: not item.stat.exists
+
+- name: Get ceph_cli
+  include_tasks: ceph_cli.yml
+  vars:
+    mount_spec: true
+
+- name: Get the ceph orchestrator status
+  command: "{{ cifmw_cephadm_ceph_cli }} orch status --format json"
+  register: ceph_orch_status
+  become: true
+
+- name: Fail if ceph orchestrator is not available
+  fail:
+    msg: "'ceph orch status' returned {{ ceph_orch_status.stdout | from_json }}"
+  when:
+    - not (ceph_orch_status.stdout | from_json).available
+
+- name: Run ceph config set for parameters before applying the spec
+  include_tasks: cephadm_config_set.yml
+
+- name: Apply spec
+  command: "{{ cifmw_cephadm_ceph_cli }} orch apply --in-file {{ cifmw_cephadm_container_spec }}"
+  register: cifmw_cephadm_apply_spec_out
+  become: true
+
+- name: Show results of spec apply
+  debug:
+    msg: "{{ cifmw_cephadm_apply_spec_out }}"
```

Comment:
We can use `var: cifmw_cephadm_apply_spec_out` instead of msg.

Reply:
done